libreoffice-24.8.yaml

```diff
@@ -0,0 +1,178 @@
+package:
+  name: libreoffice-24.8
+  version: 24.8.2.1
+  epoch: 0
+  description:
+  # https://www.libreoffice.org/about-us/licenses
+  copyright:
+    - license: GPL-3.0-only AND LGPL-3.0-only AND MPL-2.0
+  resources:
+    cpu: 65
+    memory: 32Gi
+  dependencies:
+    provides:
+      - libreoffice=${{package.full-version}}
+
+vars:
+  python-version: 3.12
+  java-version: 17
+  java-home: /usr/lib/jvm/java-17-openjdk
+
+environment:
+  contents:
+    packages:
+      - ant
+      - argon2-dev
+      - autoconf
+      - automake
+      - bash-binsh
+      - bison
+      - boost-dev
+      - build-base
+      - busybox
+      - cairo-dev
+      - coreutils
+      - cups-dev
+      - curl
+      - curl-dev
+      - dbus-dev
+      - diffutils
+      - expat-dev
+      - flex
+      - fontconfig-dev
+      - freetype-dev
+      - fribidi-dev
+      - glib-dev
+      - gnutar
+      - gperf
+      - gpgme-dev
+      - gst-plugins-base-dev
+      - gstreamer-dev
+      - gtk-3-dev
+      - harfbuzz-dev
+      - krb5-dev
+      - libassuan-dev
+      - libatk-1.0
+      - libatk-bridge-2.0
+      - libcmis-dev
+      - libice-dev
+      - liblangtag-dev
+      - libnspr-dev
+      - libnss-dev
+      - libsm-dev
+      - libtool
+      - libx11-dev
+      - libxcomposite-dev
+      - libxcursor-dev
+      - libxdamage-dev
+      - libxft-dev
+      - libxml2-dev
+      - libxrandr-dev
+      - libxrender
+      - libxslt-dev
+      - libxt-dev
+      - libxtst-dev
+      - m4
+      - mesa
+      - mesa-dev
+      - mesa-egl
+      - mesa-gl
+      - nasm
+      - openjdk-${{vars.java-version}}
+      - openjdk-${{vars.java-version}}-default-jvm
+      - openssl-dev
+      - orc-dev
+      - pango-dev
+      - patch
+      - perl
+      - pkgconf
+      - pkgconf-dev
+      - py${{vars.python-version}}-setuptools # To fix `No module named 'distutils'` after bumping to python 3.12
+      - python-${{vars.python-version}}-dev
+      - zip
+  environment:
+    JAVA_HOME: ${{vars.java-home}}
+
+pipeline:
+  - uses: git-checkout
+    with:
+      repository: https://github.com/LibreOffice/core
+      tag: libreoffice-${{package.version}}
+      expected-commit: 0f794b6e29741098670a3b95d60478a65d05ef13
+
+  # patch rather than cherry-pick. The git fetch of main takes multiple minutes.
+  - uses: patch
+    with:
+      patches: 0001-raptor-Use-without-www.patch
```

Comment:
@smoser this patch is missing i think. symlink or copy the dir from previous libreoffice?

Reply:
Doh. Thanks. I'll fix

Reply:
done.